### PR TITLE
Implementing of Fat 1376

### PR DESCRIPTION
**Pull request by IhorBohdan**
***
[FAT-1376](https://issues.folio.org/browse/FAT-1376)
***
[C737](https://foliotest.testrail.io/index.php?/cases/view/737)
***
[Allure](https://jenkins-aws.indexdata.com/job/Testing/job/Cypress_tests/2752/allure/)
***

### CODE REVIEW

File: cypress/support/fragments/orders/orderLines.js [hunk 1]
```diff
@@ -205,6 +205,23 @@ export default {
     ]);
   },
 
+  POLineInfodorPhysicalMaterialForRecieve: (orderLineTitleName) => {
+    cy.do([
+      orderLineTitleField.fillIn(orderLineTitleName),
+      orderFormatSelect.choose('Physical resource'),
```
Comment (penfriend) [inline cypress/support/fragments/orders/orderLines.js:211]: let's move this string to const variables to separate constants file which is located in the support directory

File: cypress/support/fragments/orders/orderLines.js [hunk 2]
```diff
@@ -205,6 +205,23 @@ export default {
...
+      acquisitionMethodButton.click(),
+      SelectionOption('Depository').click(),
```
Comment (penfriend) [inline cypress/support/fragments/orders/orderLines.js:213]: let's move this string to const variables to separate constants file which is located in the support directory

File: cypress/e2e/orders/receiving-and-check-in/status-is-updated-from-n-order.cy.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    Receiving.selectPOLInReceive(orderLineTitle);
+    Receiving.addPiece(caption, copyNumber, enumeration, chronology);
+    Receiving.selectPiece(caption);
+    Receiving.quickReceivePiece(enumeration);
+    Receiving.selectInstanceInReceive(orderLineTitle);
+    // inventory part
+    InventoryInstance.openHoldingsAccordion(OrdersHelper.onlineLibraryLocation);
+    ItemRecordView.findRowAndClickLink(copyNumber);
+    ItemRecordView.checkEffectiveLocation(OrdersHelper.onlineLibraryLocation);
+    ItemRecordView.checkStatus('In process');
```
Comment (penfriend) [inline cypress/e2e/orders/receiving-and-check-in/status-is-updated-from-n-order.cy.js:87]: let's move this string to const variables to separate constants file which is located in the support directory

**Review by penfriend — CHANGES_REQUESTED** [review]

**Review by TetianaParanich — APPROVED** [review]

File: cypress/support/fragments/orders/orderLines.js (hunk 1 above)
Comment (IhorBohdan) [reply cypress/support/fragments/orders/orderLines.js]: Done

File: cypress/support/fragments/orders/orderLines.js (hunk 2 above)
Comment (IhorBohdan) [reply cypress/support/fragments/orders/orderLines.js]: Done

File: cypress/e2e/orders/receiving-and-check-in/status-is-updated-from-n-order.cy.js (hunk 3 above)
Comment (IhorBohdan) [reply cypress/e2e/orders/receiving-and-check-in/status-is-updated-from-n-order.cy.js]: Done

File: cypress/support/fragments/orders/orderLines.js [hunk 4]
```diff
@@ -371,9 +389,9 @@ export default {
...
     cy.do([
       orderLineTitleField.fillIn(orderLineTitle),
-      orderFormatSelect.choose('Physical resource'),
+      orderFormatSelect.choose(ORDER_FORMAT_NAMES.PHYSICAL_RESOURCE),
       acquisitionMethodButton.click(),
-      SelectionOption('Depository').click(),
+      SelectionOption(ACQUISITION_METHOD_NAMES.DEPOSITORY).click(),
       physicalUnitPriceTextField.fillIn(physicalUnitPrice),
       quantityPhysicalTextField.fillIn('2'),
       materialTypeSelect.choose('book'),
```
Comment (penfriend) [inline cypress/support/fragments/orders/orderLines.js:397]: Please use here the variable from the constants file from the support directory

File: cypress/support/fragments/orders/orderLines.js [hunk 5]
```diff
@@ -765,10 +783,10 @@ export default {
...
-      orderFormatSelect.choose('Other'),
+      orderFormatSelect.choose(ORDER_FORMAT_NAMES.OTHER),
       acquisitionMethodButton.click(),
-      SelectionOption('Depository').click(),
-      receivingWorkflowSelect.choose('Synchronized order and receipt quantity'),
+      SelectionOption(ACQUISITION_METHOD_NAMES.DEPOSITORY).click(),
+      receivingWorkflowSelect.choose(RECEIVING_WORKFLOW_NAMES.SYNCHRONIZED_ORDER_AND_RECEIPT_QUANTITY),
       physicalUnitPriceTextField.fillIn(physicalUnitPrice),
       quantityPhysicalTextField.fillIn(quantityPhysical),
       materialTypeSelect.choose('book'),
```
Comment (penfriend) [inline cypress/support/fragments/orders/orderLines.js:792]: the same here 
Please use here the variable from the constants file from the support directory

File: cypress/support/fragments/orders/orderLines.js [hunk 6]
```diff
@@ -190,10 +191,10 @@ export default {
...
-      orderFormatSelect.choose('Physical resource'),
+      orderFormatSelect.choose(ORDER_FORMAT_NAMES.PHYSICAL_RESOURCE),
       acquisitionMethodButton.click(),
-      SelectionOption('Depository').click(),
-      receivingWorkflowSelect.choose('Synchronized order and receipt quantity'),
+      SelectionOption(ACQUISITION_METHOD_NAMES.DEPOSITORY).click(),
+      receivingWorkflowSelect.choose(RECEIVING_WORKFLOW_NAMES.SYNCHRONIZED_ORDER_AND_RECEIPT_QUANTITY),
       physicalUnitPriceTextField.fillIn(physicalUnitPrice),
       quantityPhysicalTextField.fillIn(quantityPhysical),
       materialTypeSelect.choose('book'),
```
Comment (penfriend) [inline cypress/support/fragments/orders/orderLines.js:200]: Please use here the variable from the constants file from the support directory

File: cypress/support/fragments/orders/orderLines.js [hunk 7]
```diff
@@ -205,13 +206,30 @@ export default {
...
+  POLineInfodorPhysicalMaterialForRecieve: (orderLineTitleName) => {
+    cy.do([
+      orderLineTitleField.fillIn(orderLineTitleName),
+      orderFormatSelect.choose(ORDER_FORMAT_NAMES.PHYSICAL_RESOURCE),
+      acquisitionMethodButton.click(),
+      SelectionOption(ACQUISITION_METHOD_NAMES.DEPOSITORY).click(),
+      receivingWorkflowSelect.choose(RECEIVING_WORKFLOW_NAMES.SYNCHRONIZED_ORDER_AND_RECEIPT_QUANTITY),
+      physicalUnitPriceTextField.fillIn(physicalUnitPrice),
+      quantityPhysicalTextField.fillIn('1'),
+      materialTypeSelect.choose('book'),
```
Comment (penfriend) [inline cypress/support/fragments/orders/orderLines.js:218]: Please use here the variable from the constants file from the support directory

File: cypress/support/fragments/orders/orderLines.js [hunk 8]
```diff
@@ -205,13 +206,30 @@ export default {
...
-      orderFormatSelect.choose('Physical resource'),
+      orderFormatSelect.choose(ORDER_FORMAT_NAMES.PHYSICAL_RESOURCE),
       acquisitionMethodButton.click(),
-      SelectionOption('Depository').click(),
-      receivingWorkflowSelect.choose('Synchronized order and receipt quantity'),
+      SelectionOption(ACQUISITION_METHOD_NAMES.DEPOSITORY).click(),
+      receivingWorkflowSelect.choose(RECEIVING_WORKFLOW_NAMES.SYNCHRONIZED_ORDER_AND_RECEIPT_QUANTITY),
       physicalUnitPriceTextField.fillIn(physicalUnitPrice),
       quantityPhysicalTextField.fillIn(quantityPhysical),
       materialTypeSelect.choose('book'),
```
Comment (penfriend) [inline cypress/support/fragments/orders/orderLines.js:235]: Please use here the variable from the constants file from the support directory

File: cypress/support/fragments/orders/orderLines.js [hunk 9]
```diff
@@ -229,10 +247,10 @@ export default {
...
-      orderFormatSelect.choose('Physical resource'),
+      orderFormatSelect.choose(ORDER_FORMAT_NAMES.PHYSICAL_RESOURCE),
       acquisitionMethodButton.click(),
-      SelectionOption('Depository').click(),
-      receivingWorkflowSelect.choose('Independent order and receipt quantity'),
+      SelectionOption(ACQUISITION_METHOD_NAMES.DEPOSITORY).click(),
+      receivingWorkflowSelect.choose(RECEIVING_WORKFLOW_NAMES.INDEPENDENT_ORDER_AND_RECEIPT_QUANTITY),
       physicalUnitPriceTextField.fillIn(physicalUnitPrice),
       quantityPhysicalTextField.fillIn(quantityPhysical),
       materialTypeSelect.choose('book'),
```
Comment (penfriend) [inline cypress/support/fragments/orders/orderLines.js:256]: Please use here the variable from the constants file from the support directory

File: cypress/support/fragments/orders/orderLines.js [hunk 10]
```diff
@@ -251,10 +269,10 @@ export default {
...
-      orderFormatSelect.choose('Other'),
+      orderFormatSelect.choose(ORDER_FORMAT_NAMES.OTHER),
       acquisitionMethodButton.click(),
-      SelectionOption('Depository').click(),
-      receivingWorkflowSelect.choose('Independent order and receipt quantity'),
+      SelectionOption(ACQUISITION_METHOD_NAMES.DEPOSITORY).click(),
+      receivingWorkflowSelect.choose(RECEIVING_WORKFLOW_NAMES.INDEPENDENT_ORDER_AND_RECEIPT_QUANTITY),
       physicalUnitPriceTextField.fillIn(physicalUnitPrice),
       quantityPhysicalTextField.fillIn(quantityPhysical),
       materialTypeSelect.choose('book'),
```
Comment (penfriend) [inline cypress/support/fragments/orders/orderLines.js:278]: Please use here the variable from the constants file from the support directory

File: cypress/support/fragments/orders/orderLines.js (hunk 4 above)
Comment (IhorBohdan) [reply cypress/support/fragments/orders/orderLines.js]: Done

File: cypress/support/fragments/orders/orderLines.js (hunk 6 above)
Comment (IhorBohdan) [reply cypress/support/fragments/orders/orderLines.js]: Done

File: cypress/support/fragments/orders/orderLines.js (hunk 8 above)
Comment (IhorBohdan) [reply cypress/support/fragments/orders/orderLines.js]: Done

File: cypress/support/fragments/orders/orderLines.js (hunk 9 above)
Comment (IhorBohdan) [reply cypress/support/fragments/orders/orderLines.js]: Done

File: cypress/support/fragments/orders/orderLines.js (hunk 10 above)
Comment (IhorBohdan) [reply cypress/support/fragments/orders/orderLines.js]: Done

**Review by penfriend — APPROVED** [review]

**Pull request merged**